### PR TITLE
SQSCANGHA-54 Rebranding

### PR DESCRIPTION
**Pull request by antonioaversa**
Rebrand the GitHub Action as "SonarQube Server and Cloud". Main changes are:
- the [README.md](https://github.com/SonarSource/sonarqube-scan-action/blob/antonio/rebranding/README.md)
- in the [action.yml](https://github.com/SonarSource/sonarqube-scan-action/blob/antonio/rebranding/action.yml)

Prerequisite for the deprecation of the GitHub Action for SonarCloud.

~The base of this PR is https://github.com/SonarSource/sonarqube-scan-action/pull/151, as it builds on top of it.~

~The PR is kept in draft to avoid accidental merging.~

https://github.com/SonarSource/sonarqube-scan-action/pull/151 has been merged, so this PR now targets `dev`.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,72 +62,130 @@ jobs:
...
+- `SONAR_TOKEN` – **Required** this is the token used to authenticate access to SonarQube. You can read more about security tokens in the documentation of SonarQube [Server](https://docs.sonarsource.com/sonarqube-server/latest/user-guide/managing-tokens/) and [Cloud](https://docs.sonarsource.com/sonarqube-cloud/managing-your-account/managing-tokens/). You can set the `SONAR_TOKEN` environment variable in the "Secrets" settings page of your repository, or you can add them at the level of your GitHub organization (recommended).
+- *`GITHUB_TOKEN` – Provided by Github (see [Authenticating with the GITHUB_TOKEN](https://help.github.com/en/actions/automating-your-workflow-with-github-actions/authenticating-with-the-github_token)).*
+- `SONAR_HOST_URL` – **Required** this tells the scanner where SonarQube is hosted.
+  - For SonarQube Server: you can set the `SONAR_HOST_URL` environment variable in the "Variables" settings page of your repository, or you can add them at the level of your GitHub organization (recommended).
+  - For SonarQube Cloud: it should be set to `https://sonarcloud.io`.
```
Comment (henryju) [inline README.md:156]: This is the default for new scanners, so I think you can drop this line.

File: README.md [hunk 2]
```diff
@@ -51,72 +62,130 @@ jobs:
...
+- `SONAR_ROOT_CERT` – Holds an additional root certificate (in PEM format) that is used to validate the SonarQube Server certificate. You can set the `SONAR_ROOT_CERT` environment variable in the "Secrets" settings page of your repository, or you can add them at the level of your GitHub organization (recommended).
```
Comment (henryju) [inline README.md:157]: Can also be useful for SonarQube Cloud, in case of a proxy.

**Review by henryju — APPROVED** [review]
Minot comments then LGTM

File: README.md (hunk 1 above)
Comment (antonioaversa) [reply README.md]: Oh, I didn't know, that's great! I have changed to text to reflect that.

This simplifies the migration of users from `sonarcloud-github-action` to `sonarqube-scan-action`, as, apart from the change of action, it should work out of the box without any argument or env var change, so it's basically a drop-in replacement!

File: README.md (hunk 2 above)
Comment (antonioaversa) [reply README.md]: Indeed, I have changed the text to reflect that.

**Pull request merged**